src/i18n/namespaces/welcome.ts

```diff
@@ -0,0 +1 @@
+export const welcomeI18nNamespace = { ns: "welcome" };
```

Comment:
Just for my understanding. 
Do we create on file for each namespace or eventually we might have multiple namspaces per file?

Reply:
Looks like my logic was going both ways when I pushed this. Looking again I think it makes the most sense to just have one namespace file with each namespace in it.

Thanks for calling that out, just pushed an update.